docs/static/images/Cloud-Bigtable.svg

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
I wondered about that too, but it seems that the icons are provided for download rather than on a CDN:  https://cloud.google.com/icons/